core/trino-main/pom.xml

```diff
@@ -288,6 +288,16 @@
             <artifactId>joda-time</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>net.jodah</groupId>
+            <artifactId>failsafe</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>net.minidev</groupId>
+            <artifactId>json-smart</artifactId>
```

Comment:
What is this? Why do you need a new json library?

Reply:
There is a simple fix for this which I've noted in another comment.

Reply:
I've removed this library from dependencies.